mmengine/runner/log_processor.py

```diff
@@ -271,6 +273,12 @@ def get_log_after_epoch(self,
             log_items.append(f'{name}: {val}')
         log_str += '  '.join(log_items)
 
+        # If cuda is available, the max memory occupied should be calculated.
+        if is_cuda_available():
+            max_memory = self._get_max_memory(runner)
+            log_str += f'memory: {max_memory}  '
+            tag['memory'] = max_memory
+
```

Comment:
Hi, `get_log_after_epoch` does not need to add memory to `tag` to append it to `log_str`.